src/locales/cy/translation.json

```diff
@@ -2206,8 +2206,8 @@
         }
       },
       "2FaBeforeResetPassword": {
-        "title": "Enter a security code from your authenticator app",
-        "header": "Enter a security code from your authenticator app",
+        "title": "Rhowch god diogelwch o’ch ap dilysu",
+        "header": "Rhowch god diogelwch o’ch ap dilysu",
         "info": {
           "paragraph1": "I gael cod diogelwch, agorwch yr ",
```

Comment:
I don't think this Welsh text is correct. This Slack message says it should be:
'I gael cod diogelwch, agorwch yr ap dilysydd a ddefnyddiwyd gennych i greu eich GOV.UK One Login'

https://gds.slack.com/archives/C06JNAG2GH3/p1707916743558909